### PR TITLE
feat: use image_url from survey object at surveys page

### LINKED ISSUE #705: use image_url from survey object at surveys page
The user can register an `image url` when he's creating a new survey. This image_url when it exists, it'll be display a cover photo.
image

Now, survey object has the optional field `image_url`
image

The image_url can be found in:
image

### PR DESCRIPTION
**Pull request by mizek1**
fix #705 

# Use image_url from survey object to render a cover image to survey item

## Type of changes

- [ ] **Bug fix**
- [X] **New feature**
- [ ] **Breaking change**

---

## Checklist:

- [x] My code is understandable
- [x] My code follows the styleguide of this project
- [x] My changes don't generate new warnings/errors
- [x] I have performed a self-review of my own code (build/lint/tests)

  ### Frontend

  - Components:

    - [x] I have implemented/updated unit tests
    - [ ] I have implemented/updated Storybook documentation

  - Pages:
    - [ ] I have implemented/updated E2E tests

### CODE REVIEW

File: frontend/src/components/survey-item/survey-item.test.tsx [hunk 1]
```diff
@@ -59,15 +60,19 @@ describe('SurveyItem', () => {
...
-      <SurveyItem
-        title="Survey name"
-        description="Available your instincts now"
-        numberQuestions={10}
-      />
+      <SurveyItem {...mockSurveyDefault} cover={''} />
     );
-    expect(rendered.getByTestId('random-image')).toBeTruthy();
+    const coverImageUrl = getComputedStyle(rendered.getByTestId('cover-image')).backgroundImage;
+    expect(images.includes(coverImageUrl.replace(/^url\(['"]?/, '').replace(/['"]?\)$/, ''))).toBeTruthy();
```
Comment (edimossilva) [inline frontend/src/components/survey-item/survey-item.test.tsx:75]: Can you think of some way to improve the readability of these 2 `replace` methods?
(just a suggestion, not a blocker)

**Review by edimossilva — APPROVED** [review]
LGTM!

File: frontend/src/components/survey-item/survey-item.test.tsx (hunk 1 above)
Comment (mizek1) [reply frontend/src/components/survey-item/survey-item.test.tsx:75]: Sure!

Comment (mizek1) [reply frontend/src/components/survey-item/survey-item.test.tsx]: @edimossilva please, check if it's better now.

**Pull request merged**